document/core/exec/instructions.rst

```diff
@@ -3039,17 +3039,32 @@ Throwing an exception with :ref:`tag address <syntax-tagaddr>` :math:`a`
 
 .. _exec-caughtadm:
 
-Holding a caught exception with |CAUGHTadm|
-...........................................
+Exiting a |CAUGHTadm|
```

Comment:
```suggestion
Exiting a catch clause |CAUGHTadm|
```

Reply:
Made PR #240 with this and the other changes requested.